### PR TITLE
Feat: Endpoint to use an item from inventory (out of combat)

### PR DESCRIPTION
**Pull request by SilviaPabon**
**How can I test this?**

- use endpoint `/items/use`
- send `loomie_id` to increase its level and an `item_id` (only works with unknow beverage)
- if quantity of items is over is deleted from user.items array

- endpoint docs about this endpoint were updated as well

### CODE REVIEW

File: api/models/loomies.go [hunk 1]
```diff
@@ -250,3 +250,27 @@ func InsertUserInArrayOfWildLoomie(loomie interfaces.WildLoomie, user interfaces
 
 	return err
 }
+
+func UpdateLevelOfLoomie(userId primitive.ObjectID, loomieId primitive.ObjectID) (bool, error) {
```
Comment (PedroChaparro) [inline api/models/loomies.go]: I suggest to update this function to make it "more general": You can also receive the amount of levels the Loomie will increase.

**Review by PedroChaparro — APPROVED** [review]

**Review by Woynert — CHANGES_REQUESTED** [review]
If you provide gibberish as the **loomie_id** the Unknown Beverage will be consumed (quantity decreased by one); no Loomie will be leveled up, and an **500 Internal Server Error** will be returned.

Example:

```
{
	"item_id": "643335b43df7c1be7460d7c6",
	"loomie_id": "anythinghere"
}
```

I am able to reliably reproduce this.

**Comment by SilviaPabon** [comment]
> If you provide gibberish as the **loomie_id** the Unknown Beverage will be consumed (quantity decreased by one); no Loomie will be leveled up, and an **500 Internal Server Error** will be returned.
> 
> Example:
> 
> ```
> {
> 	"item_id": "643335b43df7c1be7460d7c6",
> 	"loomie_id": "anythinghere"
> }
> ```
> 
> I am able to reliably reproduce this.

@Woynert and @PedroChaparro I think that I had fix this, please check it!

File: api/models/user.go [hunk 2]
```diff
@@ -536,3 +536,27 @@ func DecrementItemFromUserInventory(userId primitive.ObjectID, itemId primitive.
...
+func IncrementItemFromUserInventory(userId primitive.ObjectID, itemId primitive.ObjectID, quantity int) error {
+	//Update the number of items in mongo
+	filter := bson.D{{Key: "_id", Value: userId}, {Key: "items.item_id", Value: itemId}}
+	update := bson.D{
+		{
+			Key: "$inc",
+			Value: bson.D{
+				{
+					Key:   "items.$.item_quantity",
+					Value: 1,
```
Comment (PedroChaparro) [inline api/models/user.go:550]: So, you aren't using the "quantity" argument?

Comment (SilviaPabon) [reply api/models/user.go:550]: Good point, I will change it!

Comment (SilviaPabon) [reply api/models/user.go]: Done!

**Review by andres123dbh — APPROVED** [review]

**Review by Woynert — APPROVED** [review]
:clap:

**Pull request merged**